Assets/MixedRealityToolkit.Services/CameraSystem/MixedRealityCameraSystem.cs

```diff
@@ -63,12 +67,39 @@ public MixedRealityCameraProfile CameraProfile
                 return cameraProfile;
             }
         }
+        
+        /// <inheritdoc/>
+        public Camera Main
+        {
+            get
+            {
+                if (main == null || !main.isActiveAndEnabled)
+                {
+                    FindOrCreateMainCamera();
+                }
+                return main;
+            }
+        }
 
         private DisplayType currentDisplayType;
         private bool cameraOpaqueLastFrame = false;
+        private static Camera main;
 
         /// <inheritdoc />
         public override void Initialize()
+        {
+            FindOrCreateMainCamera();
+
+            // Ensure that the camera is parented under the mixed reality playspace
+            main.transform.SetParent(MixedRealityPlayspace.Transform);
+
+            // There's lots of documented cases that if the camera doesn't start at 0,0,0, things break with the WMR SDK specifically.
+            // We'll enforce that here, then tracking can update it to the appropriate position later.
+            main.transform.position = Vector3.zero;
+        }
```

Comment:
Does WMR sdk require the global camera position to be 0,0,0 or the local camera position to be 0,0,0? If we account for any preexisting parent game object transforms, we may need to make sure that the global position is zero here (i'm not sure if setting position sets a true position relative to all parent transforms).

Reply:
Not sure. This has been the default behavior up to this point so I don't want to change it without a lot of testing. Probably belongs in another PR.